### ISSUE TITLE
Persistent Issue: Transmission Settings Reset on Windows 11

### ISSUE THREAD

**Issue by ghost** [transmission/transmission#6472]
### What is the issue?

Upon rebooting my OS and initiating Transmission, the settings revert to their default values, although the added torrents are unaffected. I've tried installing Transmission using both winget and the GitHub installer, but the problem persists. This is occurring on a three-day-old Windows 11 installation.

### Which application of Transmission?

Qt app on Windows

### Which version of Transmission?

4.0.5

**Comment by Pentaphon** [comment]
I had this happen once when I first installed 4.0.0 but I was never able to reproduce it.

**Comment by Helyos96** [comment]
This happens when you reboot/shutdown windows with transmission running. If you want to save settings you have to manually close the program first.

**Comment by ae5960e8-a6fc-491f-b252-898ecf59af95** [comment]
“Used to happen” as in it's now resolved?

**Comment by drm-nz** [comment]
No, it's not fixed, it still works that way. I've just tested. You need to start the app, make your config changes, then close/reopen the app for the changes to stick.

**Comment by Pentaphon** [comment]
> You need to start the app, make your config changes, then close/reopen the app for the changes to stick.

I also notice it is especially important to restart the client at least once after you install on Windows 11 or the client has weird issues in the settings page.

**Comment by humblefalcon** [comment]
I am having this issue now.

It's frustrating because if a user happens to close and reopen the app after changing a setting in the past it can lead them to believe it's just an issue with the setting they are trying to apply now, making troubleshooting more difficult.

**Comment by dfarning** [comment]
What a weird issue. I just spent 15 minutes uninstalling and reinstalling the transmission GUI trying to see if I had screwed up the configuration setting somehow because it would not service a reboot.

Just quitting and restarting via the menu solved my problem.

**Comment by misnoma** [comment]
This issue persists as of Nov 2024.

**Comment by lukilou** [comment]
Yep, confirm that Nov 2024 still the issue is there, ```Transmission 4.0.6 (38c164933e)```

**Comment by alfa005pro** [comment]
its a serious problem

**Comment by severtheskyline** [comment]
Also happening to me on KDE Plasma with transmission-qt

**Comment by ckerr** [comment]
Duplicate of #882

**Comment by sabi0** [comment]
At the very least the Qt app could consider "OS shutdown" event as an application exit and save the settings before the shutdown?
> When Windows is about to shut down, it sends a [WM_QueryEndSession](https://docs.microsoft.com/en-us/windows/desktop/shutdown/wm-queryendsession) to all open applications.

It would be better yet to persist the settings changes when "Close" button is pressed in the Preferences dialog.
